src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -696,6 +696,14 @@ function InitializeEnvironmentalVariables(frb: FunctionsRuntimeBundle): void {
   process.env.GCLOUD_PROJECT = frb.projectId;
   process.env.FUNCTIONS_EMULATOR = "true";
 
+  const nodeVersion = parseVersionString(process.versions.node);
```

Comment:
I actually think we should use the node version they specified in `engines` not their local once since that's how the Cloud Functions will really run.

Something like:
```
const pkg = requirePackageJson(frb);
// TODO: Need to check if engines is even specified
// TODO: This version will be a string, need to parse it
const nodeVersion = pkg.engines.node;
```